code/modules/cargo/packs.dm

```diff
@@ -315,15 +318,6 @@
 					/obj/item/clothing/head/fedora/det_hat)
 	crate_name = "forensics crate"
 
-/datum/supply_pack/security/helmets
```

Comment:
You need to cleanup everything that points to these

Reply:
done! thank you